src/search/FindInFiles.js

```diff
@@ -417,15 +418,26 @@ define(function (require, exports, module) {
         }
         
         var scopeName = searchModel.scope ? searchModel.scope.fullPath : ProjectManager.getProjectRoot().fullPath,
-            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query);
+            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query),
+            searchDeferred = new $.Deferred();
         
-        return candidateFilesPromise
+        candidateFilesPromise
             .then(function (fileListResult) {
                 // Filter out files/folders that match user's current exclusion filter
                 fileListResult = FileFilters.filterFileList(filter, fileListResult);
+                fileListResult = FindUtils.getSortedFiles(fileListResult, function (file) { return file.fullPath; }, FindUtils.getOpenFilePath());
```

Comment:
The order used for find in files is the same order as the one used on the project tree (on windows), when it is fully open. Is there a way to get the files from the project tree so that we don't need to sort them?